ui/src/components/FlashMessages/index.tsx

```diff
@@ -28,7 +28,9 @@ const Message: React.FC<{ message: FlashMessage } & Pick<Props, 'onClose'>> = (p
       </div>
 
       <Tippy content="Close">
-        <MdClose className={s['close-icon']} onClick={() => p.onClose(p.message.id)} />
+        <span>
+          <MdClose className={s['close-icon']} onClick={() => p.onClose(p.message.id)} />
+        </span>
```

Comment:
I think we shouldn't use tippy for trivial things like reporting that "X" icon means close. We should use tooltips only when we explicitly want user to know something; e.g "?" icon in top left should say "No Organization" for admin.